projects/ROCKNIX/devices/SM8550/patches/linux/0200_arm64--dts--qcom--Add-AYN-QCS8550-Common.patch

```diff
@@ -109,17 +115,10 @@ index 000000000000..58d9ab342fe9
 +		compatible = "pwm-fan";
 +
 +		fan-supply = <&vdd_fan_5v0>;
-+		pwms = <&pm8550_pwm 3 40000>; /* 40us, 25kHz */
-+
-+		cooling-levels = <0 40 65 75 90 120 150 200>;
-+		#cooling-cells = <2>;
++		pwms = <&pm8550_pwm 3 100000>;
```

Comment:
100us make the regulator "vibrate" to 10kHz, which is right in the middle of humans ear spectrum